core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTest.scala

```diff
@@ -549,10 +549,10 @@ object SplitClusterDUnitTest extends SplitClusterDUnitTestObject {
 
     // also check access to complex types as string
     rs = stmt.executeQuery(
-      s"SELECT * FROM $tableName --+ complexTypeAsClob(1)")
+      s"SELECT * FROM $tableName --+ complexTypeAsJson(1)")
```

Comment:
Earlier code was converting to a custom string format in https://github.com/SnappyDataInc/snappydata/commit/20e8ef6ddcdb89742d4bff675b2dda1f72ca3990#diff-cc0d92e180612747c169d3aaceaacfa0L464 . This was not a good idea since user will not be able to parse the result easily. So that has been taken out and instead now using jackson for JSON output as is done by Spark json format handling.